backend/src/main/java/com/zeros/backend/services/Route.java

```diff
@@ -1,64 +0,0 @@
-package com.zeros.backend.services;
-
-import java.math.BigDecimal;
-import java.time.Duration;
-import java.util.Objects;
-
-public class Route {
-
-    private final BigDecimal distance;
-    private final Duration duration;
-
-    private Route(newBuilder builder) {
-        this.distance = builder.distance;
-        this.duration = builder.duration;
-    }
-
-    public BigDecimal getDistance() {
-        return distance;
-    }
-
-    public Duration getDuration() {
-        return duration;
-    }
-
-    @Override
-    public boolean equals(Object o) {
-        if (o == this) return true;
-        if (!(o instanceof Route)) {
-            return false;
-        }
-        Route route = (Route) o;
-        return Objects.equals(distance, route.distance) &&
-                Objects.equals(duration, route.duration);
-    }
-
-    @Override
-    public int hashCode() {
-        return Objects.hash(distance, duration);
-    }
-
```

Comment:
Builder is cool for immutable object but in this case we have juste 2 attributes. you can remove :+1: